### DISCUSSION TITLE
How do I save settings?

### DISCUSSION THREAD

**Discussion by ghost** [getgauge/taiko#1908] [General]
I am trying to save setting but could not do it.

**Comment by sriv** [comment]
What settings are you looking to save?

**Reply by ghost** [reply]
I am getting time error so I want to change 30000 to 120000

**Reply by sriv** [reply]
You can save some configuration by using [`setConfig`](https://docs.taiko.dev/api/setconfig/) api. this can be part of your script

**Comment by shortstacked** [comment]
I've added a PR that will allow it to be pulled from process.env which would work through Gauge variables or dotenv.

https://github.com/getgauge/taiko/pull/2013
